capstoneteamb/capstone360
# Review page update

To test:

Python functionality:
go to /review/
Fill out fields
Check the database to see if the input you entered is what shows up (I can provide a more extensive guide to testing this, but this is predominately an update to the HTML template and so python functionality should not be greatly changed)

you can update the student to review by changing the id in the get id method in form.py

HTML Template:
-Students should appear on the columns, review fields on the rows
-Check how the browser handles the table at low resolution (does everything squish together and overlap?)
-Do you see the reviewer's name at the top of the page?
